extra/1-currency-conversion.js

```diff
@@ -15,7 +19,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  price = price * 5.7 * 0.99;
+  return parseFloat(price.toFixed(2));
+}
```

Comment:
Please look into pure functions and argument mutation. 

It's considered a bad practice for a function to have side effects. Can you think of which side effect this function has and why would it be bad practice?

Reply:
only thing came my mind is "5.7" and "0.99" must have been const